web/src/main/java/org/mskcc/cbio/portal/web/ProxyController.java

```diff
@@ -86,6 +86,25 @@ public void setEnableOncokb(Boolean property) {
         }
         this.enableOncokb = property;
     }
+
+  private String civicURL;
+  @Value("${civic.url:https://civic.genome.wustl.edu/api/}")
+  public void setCivicURL(String property) {
+      if (property.isEmpty()) {
```

Comment:
It could still be an empty value if the configured value is. But indeed it may be a bit much to check for that.